app_dart/lib/src/request_handlers/scheduler/batch_backfiller.dart

```diff
@@ -71,12 +71,16 @@ class BatchBackfiller extends RequestHandler {
       final FullTask task = taskColumn.first;
       final CiYaml ciYaml = await scheduler.getCiYaml(task.commit);
       final List<Target> ciYamlTargets = ciYaml.postsubmitTargets;
+
+      final List<Target> filteredCiYamlTargets = removeNonBackFillTargets(ciYamlTargets);
```

Comment:
can we move the logic to `ciYaml`, and directly obtain the list with `ciYaml.backfillTargets`?

Reply:
Sure. I can do that.